tests/Jellyfin.Server.Implementations.Tests/Sorting/AiredEpisodeOrderComparerTests.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Emby.Server.Implementations.Sorting;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Entities.Movies;
+using MediaBrowser.Controller.Entities.TV;
+using Xunit;
+
+namespace Jellyfin.Server.Implementations.Tests.Sorting
+{
+    public class AiredEpisodeOrderComparerTests
+    {
+        [Theory]
+        [ClassData(typeof(EpisodeBadData))]
+        public void AiredEpisodeOrderCompareErrorTest(BaseItem x, BaseItem y)
```

Comment:
Names should be very self-explanatory.
```suggestion
        public void Compare_GivenNull_ThrowsArgumentNullException(BaseItem x, BaseItem y)
```